lib/httparty/request.rb

```diff
@@ -305,12 +305,9 @@ def handle_deflation
 
     def response_redirects?
       case last_response
-      when Net::HTTPMultipleChoice, # 300
-           Net::HTTPMovedPermanently, # 301
-           Net::HTTPFound, # 302
-           Net::HTTPSeeOther, # 303
-           Net::HTTPUseProxy, # 305
-           Net::HTTPTemporaryRedirect
+      when Net::HTTPNotModified # 304
+        false
+      when Net::HTTPRedirection
```

Comment:
Correct. Also if Ruby doesn't have a specific class for the whole status code, it will just use `HTTPRedirection`. The HTTP spec also says that any unrecognized 3xx codes should be treated as 300. So other than 304, it's correct to assume that all 3xx codes should result in a redirection,